db/case.ts

```diff
@@ -0,0 +1,190 @@
+import { db, pgp } from './connection-pool';
+import { getPaginationElements, retrieveCategories } from '../controllers/helpers';
+import { Contact } from './contact';
+import pgPromise from 'pg-promise';
```

Comment:
The pgp instance provides access to the `helpers` namespace and a few other APIs, I think you need to get them from the `pgp` instance because you instantiate that with some global configuration options that affect how it generates SQL with these APIs